### DISCUSSION TITLE
LXC 固件无法打开web页面

### DISCUSSION THREAD

**Discussion by wsppiup** [coolsnowwolf/lede#12209] [Bugs / 问题反馈]
和以前编译的方法一样，只是勾选tar.gz 。编译出的其他固件正常，PVE用的tar.gz创建好CT后就无法打开web页面了。以前旧版本是可以的。现在又什么办法解决吗？

**Comment by 1085106680** [comment]
看看这个
https://github.com/coolsnowwolf/lede/issues/11546

**Reply by wsppiup** [reply]
谢谢
